test/unit/Lexer-spec.js

```diff
@@ -776,6 +776,41 @@ paragraph
         });
       });
 
+      it('escaped punctuation inside emphasis', () => {
```

Comment:
May be useful to also have a new test in `tests/spec/new`  just to make sure the whole markdown pipeline is working to get the correct HTML output. This test could include some more devious cases, like different numbers of backslashes, and multiple escaped characters in a row.

Reply:
Done.